Frontend/src/pages/home/components/Hero.jsx

```
@@ -79,15 +99,14 @@ function Hero() {
             get retailer insights, and ensure fast delivery while saving money.
           </motion.p>
 
-          {/* Buttons */}
           <motion.div
             className="mt-8 flex flex-col sm:flex-row items-center gap-4"
             variants={itemVariants}
           >
-            <button className="w-full sm:w-auto inline-flex items-center justify-center rounded-full bg-primary px-6 py-3 text-sm font-medium shadow-md text-primary-foreground hover:bg-primary-hover">
-              Get started
+            <button className="w-full sm:w-auto inline-flex items-center justify-center rounded-full bg-primary px-6 py-3 text-sm font-medium text-primary-foreground hover:bg-primary-hover">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)